### ISSUE TITLE
Compile warning

### ISSUE THREAD

**Issue by jfinzel** [HypoPG/hypopg#10]
```
hypopg.c: In function ‘hypopg_get_indexdef’:
hypopg.c:1601:21: warning: ‘entry’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  if (!entry || entry->oid != indexid)
                     ^
```

Can this be addressed?

**Comment by rjuju** [comment]
Hi,

Which compiler and which version are you using?  I can't reproduce this warning with either gcc or clang (however clang found another uninitialized variable issue).

I just pusehd b77b1857e9ebcf4adfe9cbc0b887df81f33f0290, does it fix the warning?

**Comment by jfinzel** [comment]
gcc (Debian 4.7.2-5) 4.7.2

I will try to repull and see if it changes.

**Comment by jfinzel** [comment]
@rjuju that worked, thank you.  Is this available via curl?  I had to git clone your master branch to compile it.  The file at https://github.com/dalibo/hypopg/archive/1.1.0.tar.gz does not have your change and I didn't see a 1.1.1.

**Comment by rjuju** [comment]
@jfinzel ok, great.  Thanks for confirming and reporting the issue!

You can always download latest HEAD via https://github.com/dalibo/hypopg/archive/master.zip

I didn't make a new release yet, since it's the first commit since last release and there are other minor changes I'll do shortly and I want to avoid unnecessary work on people who package this extension.

**Comment by rjuju** [comment]
@jfinzel I just released v 1.1.1.  Sorry it took some times, but the good news is that v1.1.1 is now available as a debian package on the pgdg repository!
